Sources/WalletConnectPairing/Pair.swift

```diff
@@ -0,0 +1,31 @@
+import Foundation
+import WalletConnectRelay
+import Combine
+
+extension Pair {
+    struct Config {
+        let metadata: AppMetadata
+    }
+}
+
+public class Pair {
```

Comment:
@flypaper0 what do you think of the class name? Pairing was already taken.

Reply:
I suggest to make sdk name - Pairing, and old Pairing object - Pair

Reply:
`Pair` is a verb, and does not fit to ba a name of model object in my opinion 🤔